### PR TITLE
{postgres} Backtick param values enclosed in angle brackets to render correctly in docs

### PR DESCRIPTION
**Pull request by nachoalonsoportillo**
**Related command**
az postgres

**Description**
#29616 reports that some examples in the docs are missing the parameter values for --public-network-access | --public

### CODE REVIEW

**Comment by yonzhan** [comment]
postgres

**Review by jiasli — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/rdbms/_params.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def _complex_params(command_group):  # pylint: disable=too-many-statements
...
             c.argument('minimal_tls_version', arg_type=get_enum_type(['TLS1_0', 'TLS1_1', 'TLS1_2', 'TLSEnforcementDisabled']), options_list=['--minimal-tls-version'], help='Set the minimal TLS version for connections to server when SSL is enabled. Default is TLSEnforcementDisabled.', validator=tls_validator)
-            c.argument('public_network_access', options_list=['--public-network-access', '--public'], help='Enable or disable public network access to server. When disabled, only connections made through Private Links can reach this server. Allowed values are : Enabled, Disabled, all, 0.0.0.0, <SingleIP>, <StartIP-DestinationIP>. Default is Enabled.')
+            c.argument('public_network_access', options_list=['--public-network-access', '--public'], help='Enable or disable public network access to server. When disabled, only connections made through Private Links can reach this server. Allowed values are : Enabled, Disabled, all, 0.0.0.0, `<SingleIP>`, `<StartIP-DestinationIP>`. Default is Enabled.')
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/rdbms/_params.py:63]: Consider quoting `Enabled`, `Disabled`, `all`, `0.0.0.0` with backticks too.

Comment (nachoalonsoportillo) [reply src/azure-cli/azure/cli/command_modules/rdbms/_params.py:63]: `

Comment (nachoalonsoportillo) [reply src/azure-cli/azure/cli/command_modules/rdbms/_params.py:63]: Done.

File: src/azure-cli/azure/cli/command_modules/rdbms/_params.py [hunk 2]
```diff
@@ -60,7 +60,7 @@ def _complex_params(command_group):  # pylint: disable=too-many-statements
...
             c.argument('minimal_tls_version', arg_type=get_enum_type(['TLS1_0', 'TLS1_1', 'TLS1_2', 'TLSEnforcementDisabled']), options_list=['--minimal-tls-version'], help='Set the minimal TLS version for connections to server when SSL is enabled. Default is TLSEnforcementDisabled.', validator=tls_validator)
-            c.argument('public_network_access', options_list=['--public-network-access', '--public'], help='Enable or disable public network access to server. When disabled, only connections made through Private Links can reach this server. Allowed values are : Enabled, Disabled, all, 0.0.0.0, <SingleIP>, <StartIP-DestinationIP>. Default is Enabled.')
+            c.argument('public_network_access', options_list=['--public-network-access', '--public'], help='Enable or disable public network access to server. When disabled, only connections made through Private Links can reach this server. Allowed values are : `Enabled`, `Disabled`, `all`, 0.0.0.0, `<SingleIP>`, `<StartIP-DestinationIP>`. Default is `Enabled`.')
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/rdbms/_params.py:63]: Shouldn't `0.0.0.0` be quoted too?

Comment (nachoalonsoportillo) [reply src/azure-cli/azure/cli/command_modules/rdbms/_params.py:63]: Sorry. Missed that one. Done now.

**Review by evelyn-ys — APPROVED** [review]

**Pull request merged**